readme.md

```diff
@@ -94,7 +115,7 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+OAuth allows an application to hand off the responsibilities of signing up/logging in to a website to a third-party. It allows an application to authenticate users without having them create a password (similar to a valet key, which gives the valet limited access and ability to drive a car).
```

Comment:
:+1: Also the permission the user gives our app to access the user's account has limited access rights